terraform/github.py

```diff
@@ -133,7 +137,10 @@ def status(self, status: str) -> None:
         self._update_comment()
 
     def _update_comment(self):
-        comment = f'{self._comment_identifier}\n```\n{self.plan}\n```'
+        comment = (f'{self._comment_identifier}\n\n'
+                   f'```{self.plan}\n```\n'
+                   f'{self.build_url}')
```

Comment:
A terraform comment is identifier + plan + status. 
Status updates already include the build_url, but initial plans don't have a status. Can you only append it if the status is None?